packages/arb-token-bridge-ui/src/util/viem.ts

```diff
@@ -0,0 +1,8 @@
+import { Chain } from 'viem/chains'
+import { createPublicClient, http } from 'viem'
+
+export const createArbPublicClient = (chain: Chain) =>
```

Comment:
If we're not reusing it anywhere, should we just directly use `createPublicClient` in code instead of creating new file?

Reply:
if we want to migrate to viem, we'll need to reuse it
if we don't need it after refactoring, we can remove this